wpilib/wpilib/sendablechooser.py

```diff
@@ -5,8 +5,12 @@
 # must be accompanied by the FIRST BSD license file in the root directory of
 # the project.
 # ----------------------------------------------------------------------------
+from typing import TypeVar
+
+V = TypeVar("V")
```

Comment:
This type variable isn't particularly useful if it doesn't tie any types together.

If this is intended to tie together the argument and return types for SendableChooser's methods, you might also want to make this type variable covariant.

(i.e. if I have a SendableChooser of SpeedControllers, I should be able to put in a Spark, get it back, and treat it as merely a SpeedController.)

Reply:
Wouldn't a spark fall under SpeedController since it's a subclass? I add a SpeedController object (which happens to be a spark), and I get back a speedcontroller object (that is *also* a spark)

Reply:
We appear to be in violent agreement with each other.

Reply:
Oh I didn't realize python had a special marking for this type of behavior. All these small quirks have made me wonder if I should've read through pep484 before implementing it...🤔